app/services/container_dashboard_service.rb

```diff
@@ -218,6 +220,46 @@ def daily_network_metrics
     end
   end
 
+  def fill_daily_pod_metrics(metrics, pod_create_trend, pod_delete_trend)
+    metrics.each do |m|
+      timestamp = m.timestamp.strftime("%Y-%m-%d")
+
+      pod_create_trend[timestamp] += m.stat_container_group_create_rate.to_i
+      pod_delete_trend[timestamp] += m.stat_container_group_delete_rate.to_i
+    end
+  end
+
+  def daily_pod_metrics
+    daily_pod_create_trend = Hash.new(0)
+    daily_pod_delete_trend = Hash.new(0)
+
+    fill_daily_pod_metrics(daily_provider_metrics,
+                           daily_pod_create_trend, daily_pod_delete_trend)
+
+    if daily_provider_metrics.any?
```

Comment:
Could you use `daily_pod_create_trend.any?` instead?
Or maybe `daily_pod_create_trend.any? || daily_pod_delete_trend.any?`

I prefer the first, but you know your logic and the second one may be required.